.detoxrc.json

```diff
@@ -0,0 +1,47 @@
+{
+    "testRunner": "jest",
+    "runnerConfig": "e2e/config.json",
+    "skipLegacyWorkersInjection": true,
+    "devices": {
+        "emulator": {
+            "type": "android.emulator",
+            "device": {
+                "avdName": "Pixel_3a_API_30_x86"
+            }
+        },
+
+        "simulator": {
+            "type": "ios.simulator",
+            "device": {
+                "type": "iPhone 13 Pro"
+            }
+        }
+    },
+    "apps": {
+        "android.debug": {
+            "type": "android.apk",
+            "binaryPath": "android/app/build/outputs/apk/debug/app-debug.apk",
+            "build": "cd android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd .."
```

Comment:
We should try to make a release build, we can do that by un-commenting a line in the Gradle file so the app is signed with a test key, which is ok.